autoload/jedi.vim

```diff
@@ -93,7 +93,7 @@ function! s:init_python()
 
         " Add a warning in case the auto-detected version is not available,
         " usually because of a missing neovim module in a VIRTUAL_ENV.
-        if has('nvim')
+        if has('nvim') && (!exists('g:python3_host_prog') && !exists('g:python_host_prog'))
```

Comment:
@blueyed @davidhalter Thanks for the comments. I put this condition here rather line 90 since line 90 checks conditions for Vim, not Neovim as it states `if !has('nvim')`.